### PR TITLE
Fix 2.7 tests

### PR DESCRIPTION
**Pull request by sentrivana**
The Python 2.7 common test suite fails to execute properly, leading the CI to think the tests passed and unfortunately also masking multiple actual test failures.

Fixes https://github.com/getsentry/sentry-python/issues/2139

### CODE REVIEW

**Comment by sentrivana** [comment]
Hey @Zylphrex, could you please take a look at the failing profiler tests for py 2.7 ([here](https://github.com/getsentry/sentry-python/actions/runs/5187994424/jobs/9351036286?pr=2145) and [here](https://github.com/getsentry/sentry-python/actions/runs/5187994409/jobs/9351055453?pr=2145))?

TL;DR why this is coming up now: the CI wasn't actually executing the `common`/`gevent` test suites for py 2.7 until recently and a handful of the tests were actually failing without us noticing.

I'll take care of the rest but don't know much about the profiler.

**Comment by Zylphrex** [comment]
> Hey @Zylphrex, could you please take a look at the failing profiler tests for py 2.7 ([here](https://github.com/getsentry/sentry-python/actions/runs/5187994424/jobs/9351036286?pr=2145) and [here](https://github.com/getsentry/sentry-python/actions/runs/5187994409/jobs/9351055453?pr=2145))?
> 
> TL;DR why this is coming up now: the CI wasn't actually executing the `common`/`gevent` test suites for py 2.7 and a handful of the tests were actually failing without us noticing.
> 
> I'll take care of the rest but don't know much about the profiler.

You can disable the profiler tests for 2.7 as it's only meant to work with >3.

File: tests/test_scrubber.py [hunk 1]
```diff
@@ -105,11 +105,9 @@ def test_breadcrumb_extra_scrubbing(sentry_init, capture_events):
...
     }
 
-    assert event["_meta"] == {
-        "extra": {"auth": {"": {"rem": [["!config", "s"]]}}},
-        "breadcrumbs": {
-            "values": {"0": {"data": {"password": {"": {"rem": [["!config", "s"]]}}}}}
-        },
+    assert event["_meta"]["extra"]["auth"] == {"": {"rem": [["!config", "s"]]}}
+    assert event["_meta"]["breadcrumbs"] == {
+        "values": {"0": {"data": {"password": {"": {"rem": [["!config", "s"]]}}}}}
```
Comment (sentrivana) [inline tests/test_scrubber.py:110]: Changed this because in py2.7 there's an additional `'sys.argv': {'': {'len': 11}}}}` in `event["_meta"]["extra"]`, I'm guessing from [here](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/integrations/argv.py). Unsure why this only happens on 2.7 but it doesn't seem to be something that needs to be fixed.

File: tests/test_scrubber.py [hunk 2]
```diff
@@ -124,8 +122,8 @@ def test_span_data_scrubbing(sentry_init, capture_events):
 
     (event,) = events
     assert event["spans"][0]["data"] == {"password": "[Filtered]", "datafoo": "databar"}
-    assert event["_meta"] == {
-        "spans": {"0": {"data": {"password": {"": {"rem": [["!config", "s"]]}}}}}
+    assert event["_meta"]["spans"] == {
```
Comment (sentrivana) [inline tests/test_scrubber.py:125]: Same here ^

**Review by antonpirker — APPROVED** [review]
lgtm
